README.md

```diff
@@ -1,8 +1,8 @@
 OpenEmbedded/Yocto BSP layer for NVIDIA Jetson Modules
 ======================================================
 
-Jetson Linux release: R35.3.1
-JetPack release:      5.1.1
+Jetson Linux release: R35.4.1\
```

Comment:
There's a stray backslash at the end there

Reply:
This is needed for new line

Reply:
Let's confine this to the minimal update for the new L4T version and postpone the beautification of the markdown rendering for a different commit, OK?